docs/hosts.md

```diff
@@ -0,0 +1,273 @@
+
+# Registry Configuration - Introduction
+
+Configuring registries will be done by specifying (optionally) a `hosts.toml` file for
+each desired registry host in a configuration directory. **Updates under this directory
+do not require restarting the containerd daemon.**
```

Comment:
> Updates under this directory do not require restarting the containerd daemon.

Is the file evaluated every time containerd pulls images?

Reply:
Yes